semantic-conventions/src/opentelemetry/semconv/model/semantic_attribute.py

```diff
@@ -275,17 +282,21 @@ def is_simple_type(attr_type: str):
         return attr_type in (
             "string",
             "string[]",
-            "number",
-            "number[]",
+            "int",
+            "int[]",
+            "double",
+            "double[]",
             "boolean",
             "boolean[]",
```

Comment:
If you make the dictionary in type_mapper a (static) constant, you can just do `if attr_type in that_dict` (`in` checks the keys with a dictionary right-hand side)